deployment/docker-swarm/external-db/docker-compose.local-overrides.yml

```diff
@@ -14,6 +14,10 @@ services:
 #      - ALERT_DB_HOST=alertdb
 #      - ALERT_DB_PORT=5432
 #      - ALERT_DB_NAME=alertdb
+#      - ALERT_DB_SSL_MODE=require
```

Comment:
Is using "require" a new development? I thought "allow" was the preferred default?

Reply:
'Allow' is still the default. This setting it commented out meaning no value results in 'allow'. If they uncomment, this sets it to 'require' changing the default